shared/actions/team-building.tsx

```diff
@@ -141,7 +141,11 @@ export function filterForNs<S, A, L, R>(
   }
 }
 
+const makeCustomResetStore = () =>
+  TeamBuildingTypes.allowedNamespace.map(namespace => TeamBuildingGen.createTbResetStore({namespace}))
```

Comment:
not strictly necessary to emit it for each namespace since the reducer happens to check before the namespace check but that seems brittle. doesn't hurt for this to go for each one